.github/workflows/nix.yaml

```
@@ -146,19 +140,49 @@ jobs:
+
+  create-manifest:
+    needs: [push-docker, build]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create and push manifest
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo "$GITHUB_TOKEN" | docker login ghcr.io -u "${{ github.actor }}" --password-stdin
+          IMAGE_NAME="${{ needs.build.outputs.docker-image-name }}"
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)